doc/lightning-cli.1.md

```diff
@@ -25,6 +25,7 @@ Sets configuration file (default: **lightning-dir**/*config* ).
  **--mainnet**
  **--testnet**
  **--signet**
+ **--regtest**
```

Comment:
This option does not exist though!  (Maybe it should, but it doesn't!)

Reply:
Whoops!

I guess the CLI happily ignores this option because we've been using it for [this docker build](https://github.com/lnbook/lnbook/tree/develop/code/docker/c-lightning) which enables the option through the conf file.

Reply:
It's just the shorthand `--regtest` that isn't available, `--network regtest` works fine :+1: